lib/utils/merge/create-merge-message.ts

```diff
@@ -0,0 +1,180 @@
+import chalk from 'chalk'
+import { ContentTypeProps } from 'contentful-management'
+import { orderBy, truncate } from 'lodash'
+import { ChangesetItem, UpdateChangesetItem } from './types'
+import {
+  fieldChange,
+  fieldIndex,
+  getChangeKey,
+  getFieldIdForIndex
+} from './utils'
+
+type MessageChangeType = 'add' | 'delete' | 'update'
+
+export type Message = {
+  changeType: MessageChangeType
+  filterStr: string
+  level: number
+  messages: string[]
+}
+
+type KnownOperations = 'add' | 'replace' | 'move' | 'copy' | 'remove' | 'test'
+
+const filterBase = (item: ChangesetItem): string => {
+  return `contentTypes/${item.entity.sys.id}`
+}
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const patchOperationToChangeType: Record<KnownOperations, MessageChangeType> = {
+  replace: 'update',
+  move: 'update',
+  copy: 'update',
+  remove: 'delete',
+  add: 'add'
+}
+
+const valueTypeFormatter = (value: string | number | boolean) => {
+  if (typeof value == 'string') {
```

Comment:
Could we use `===` here instead?

Reply:
that must have been slipped, thanks for the finding!